frontend/workflows/projectCatalog/src/index.tsx

```diff
@@ -0,0 +1,26 @@
+import type { BaseWorkflowProps, WorkflowConfiguration } from "@clutch-sh/core";
+
+export interface WorkflowProps extends BaseWorkflowProps {}
+
+const register = (): WorkflowConfiguration => {
+  return {
+    developer: {
+      name: "Lyft",
+      contactUrl: "mailto:hello@clutch.sh",
+    },
+    path: "catalog",
+    group: "Catalog",
+    displayName: "Project Catalog",
+    routes: {
+      catalog: {
+        path: "/",
+        displayName: "Project Catalog",
+        description: "Project Catalog",
```

Comment:
Can you go ahead and update this description to something a little more verbose?

Reply:
Updated to "A searchable catalog of services"